src/firewall/core/rich.py

```diff
@@ -417,6 +426,12 @@ def _import_from_string(self, rule_str):
                     in_elements.pop() # protocol
                 else:
                     raise FirewallError(errors.INVALID_RULE, "invalid 'protocol' element")
+            elif in_element == 'tcp-mss-clamp':
+                if attr_name == 'value':
+                    self.element = Rich_Tcp_Mss_Clamp(attr_value)
+                    in_elements.pop()   #tcp-mss-clamp
+                else:
```

Comment:
I think `value` is optional, right? This will throw an error for the following:
```
firewall-cmd --add-rich-rule='rule family=ipv4 tcp-mss-clamp'
```

You should add test coverage for this.